tests/cli/test_cli_utils.py

```diff
@@ -39,3 +40,26 @@ def test_is_valid_project(self):
         assert is_valid_project(None, None, "moz-fx-data-shared-prod")
         with pytest.raises(BadParameter):
             assert is_valid_project(None, None, "not-existing")
+
+    def test_table_matches_patterns(self):
+        assert not table_matches_patterns(
+            table="telemetry_live.main_v4",
+            pattern=["telemetry_live.main_v4", "telemetry_live.event_v4"],
+            invert=True,
+        )
```

Comment:
This is the case that's fixed here. This error was causing tables that we wanted to be excluded in https://github.com/mozilla/telemetry-airflow/blob/main/dags/copy_deduplicate.py#L81-L85 to get processed in `copy_deduplicate_all`.

Reply:
😱  Thanks for fixing this!